### PR TITLE
Uses waitForAsync instead of async

### PR DESCRIPTION
**Pull request by alfonsserra**

### CODE REVIEW

File: projects/systelab-login/src/lib/role-directives/allowed-roles.directive.spec.ts [hunk 1]
```diff
@@ -56,7 +56,7 @@ describe('AllowedRolesDirective', () => {
 				]
 		})
 			.compileComponents();
-	}));
+	});
 
 	beforeEach(() => {
```
Comment (DaniMartinez-Gonzalez) [inline projects/systelab-login/src/lib/role-directives/allowed-roles.directive.spec.ts:61]: Do we need to put async on this beforeEach as we have on line 45? Thanks!

File: projects/systelab-login/src/lib/role-directives/forbidden-roles.directive.spec.ts [hunk 2]
```diff
@@ -63,7 +63,7 @@ describe('ForbiddenRolesDirective', () => {
 				]
 		})
 			.compileComponents();
-	}));
+	});
 
 	beforeEach(() => {
```
Comment (DaniMartinez-Gonzalez) [inline projects/systelab-login/src/lib/role-directives/forbidden-roles.directive.spec.ts:68]: I have the same doubt as before about adding or not the async on this beforeEach. Thanks!

File: projects/systelab-login/src/lib/role-directives/allowed-roles.directive.spec.ts (hunk 1 above)
Comment (alfonsserra) [reply projects/systelab-login/src/lib/role-directives/allowed-roles.directive.spec.ts:61]: The TestBed.compileComponents method asynchronously compiles all components configured in the testing module.
Please see https://angular.io/guide/testing-components-scenarios#the-async-beforeeach

File: projects/systelab-login/src/lib/role-directives/forbidden-roles.directive.spec.ts (hunk 2 above)
Comment (alfonsserra) [reply projects/systelab-login/src/lib/role-directives/forbidden-roles.directive.spec.ts:68]: The TestBed.compileComponents method asynchronously compiles all components configured in the testing module.
Please see https://angular.io/guide/testing-components-scenarios#the-async-beforeeach

File: projects/systelab-login/src/lib/role-directives/allowed-roles.directive.spec.ts (hunk 1 above)
Comment (DaniMartinez-Gonzalez) [reply projects/systelab-login/src/lib/role-directives/allowed-roles.directive.spec.ts:61]: Thanks!

**Review by DaniMartinez-Gonzalez — APPROVED** [review]

**Review by atzurk — APPROVED** [review]
I'm OK with this PR, it resembles the e2e dealing with async functions...

File: projects/showcase/src/app/app.component.spec.ts [hunk 3]
```diff
@@ -13,7 +13,7 @@ import { ShowcaseModule } from './showcase/showcase.module';
 import { APP_BASE_HREF } from '@angular/common';
 
 describe('AppComponent', () => {
-	beforeEach(async(() => {
+	beforeEach(async () => {
 		TestBed.configureTestingModule({
```
Comment (oGalceran1) [inline projects/showcase/src/app/app.component.spec.ts:17]: In the other cases, there is an await before the configureTestingModule, but not here.
Is this correct?

**Pull request merged**